roles/prepare_sys/tasks/host.yaml

```diff
@@ -34,7 +34,7 @@
                         hostvars[_node]['ansible_all_ipv6_addresses']                    -%}
       {%     set _clust_name = hostvars[_node]['neerali_ceph_cluster_name']              -%}
       {%     for _ip in _ips                                                             -%}
-      {%         if _ip is ansible.utils.in_network neerali_ceph_public_network          -%}
+      {%         if _ip is ansible.utils.in_network(neerali_ceph_public_network)         -%}
```

Comment:
Eitherway works

Reply:
yeah just now tested with previous syntax and that worked. Reverted my changes.